### DISCUSSION TITLE
magic comment to prevent auto compile

### DISCUSSION THREAD

**Discussion by frey-iutsd** [James-Yu/LaTeX-Workshop#4413] [Q&A]
Is there a kind of magic comment that I can put at the top of a file to prevent it being automatically compiled on save? I often have part of a document in a "prelude" file which can't be compiled individually, so that would be helpful.

**Comment by James-Yu** [comment]
There's no such a thing. You may want to play with config `rootFile.indicator` and put that line to different file to cope with your need.
